janus_messages/src/lib.rs

```diff
@@ -126,7 +119,7 @@ impl Time {
     /// Construct a [`Time`] representing the instant that is a given number of seconds after
     /// January 1st, 1970, at 0:00:00 UTC (i.e., the instant with the Unix timestamp of
     /// `timestamp`).
-    pub fn from_seconds_since_epoch(timestamp: u64) -> Self {
```

Comment:
Matching the `0.1` version: we should drop the `chrono` dependency from `janus_messages`.